docs/user/alerting/action-variables.asciidoc

```diff
@@ -5,405 +5,230 @@
-`kibana.alert.reason`:: The reason of the alert (generated with the rule conditions). preview:[]
-`kibana.alert.start`:: Datetime stamp of alert start. preview:[]
-`kibana.alert.status`:: Alert status (for example, active or OK). preview:[]
+`_id`::
+(string) For example, `27ace796-1dfb-4db9-9ce1-7a10472fd461`.
+// TBD: Is this the same as the alert.uuid?
+`_index`::
+(string) For example, `.internal.alerts-stack.alerts-default-000001`.
+// TBD: The index that contains the alert data.
+`@timestamp`::
+(string) For example, `2024-04-30T02:30:11.254Z`.
+// TBD: The timestamp that the alert was added to the index?
+`event.action`::
+(string) For example: `active`.
+// TBD: Is this the same as the alert.status?
+`event.kind`::
+(string) For example: `signal`.
+// TBD: The type of event (are there types other than signal?)
+`kibana.alert.action_group`::
+(string) The ID of the action group of the alert that scheduled the action. For example, `query matched`.
+// TBD: Is this a repeat of alert.actionGroup?
+`kibana.alert.consecutive_matches`::
+(integer)
+// TBD: The number of consecutive runs that generated the alert?
+// How is this different than alert.consecutiveMatches?
+`kibana.alert.duration.us`::
+(integer) For example, `5670590000`.
+// TBD: The duration of the alert in microseconds?
+`kibana.alert.end`::
+(string) Datetime stamp of alert end. preview:[]
+// TBD: Is this when the alert recovered?
+`kibana.alert.evaluation`::
+(object)
+// TBD: The conditions that were met to generate the alert?
+`kibana.alert.flapping`::
+(boolean) A flag on the alert that indicates whether the alert status is changing repeatedly. preview:[]
+`kibana.alert.flapping_history`::
+(array of boolean)
+// TBD: The twenty most recent flapping states?
+`kibana.alert.instance.id`::
+(string) Identifier for the source that generates the alert. For example, `query matched`. preview:[]
+`kibana.alert.maintenance_window_ids`::
+(array of strings) The identifiers for maintenance windows that affect the alert.
+`kibana.alert.reason`::
+(string) The reason for the alert (generated with the rule conditions). preview:[]
+`kibana.alert.rule`::
+(object) Details about the rule.
+//TBD How is this different than the other rule object?
+`kibana.alert.start`::
+(string) Datetime stamp of alert start. For example, `2024-04-30T00:55:40.664Z`. preview:[]
+`kibana.alert.status`::
+(string) The alert status. For example, `active` or `OK`. preview:[]
+// TBD: What is the meaning of "OK" here? Should it be active, recovered, and untracked?
+`kibana.alert.time_range.gte`::
+(string) For example `2024-04-30T00:55:40.664Z`.
+//TBD This is presumably "greater than or equal to" but what's the purpose?
+`kibana.alert.time_range.lte`::
+(string) For example `2024-04-30T00:55:40.664Z`.
+//TBD Ditto "less than or equal to" but why?
+`kibana.alert.title`::
+(string) For example, `Document count is 3 in the last 5d. Alert when greater than 0`.
+// TBD: A generated title for the alert?
```

Comment:
It looks like this field will be set by some of the alerting rule types; I can see index threshold set it to `alert {name} group {group} met threshold` for active alerts.